### PR TITLE
changes related to InstallTools 3sdk update

### PR DESCRIPTION
**Pull request by dmahmouali**
- some naming refactoring
- import checks 
issue: https://github.com/threefoldtech/jumpscaleX_core/issues/838

### CODE REVIEW

File: ThreeBotPackages/zerobot/admin/actors/notifications.py [hunk 1]
```diff
@@ -8,4 +12,7 @@ def _init(self, **kwargs):
 
     @j.baseclasses.actor_method
     def check_new_release(self, schema_out=None, user_session=None):
-        return j.data.serializers.json.dumps(Tools.get_newest_version())
+        if Tools.is_latest_release():
```
Comment (grimpy) [inline ThreeBotPackages/zerobot/admin/actors/notifications.py:15]: shouldnt this be if not latestversion?

File: ThreeBotPackages/zerobot/admin/actors/notifications.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
 from Jumpscale import j
-from threesdk.InstallTools import Tools
+
+try:
+    from threesdk.InstallTools import Tools
+except:
+    from .core.InstallTools import Tools
```
Comment (waleedhammam) [inline ThreeBotPackages/zerobot/admin/actors/notifications.py:6]: should be `from Jumpscale.core.InstallTools import Tools`
should be full import . is relative to install folder

Comment (waleedhammam) [reply ThreeBotPackages/zerobot/admin/actors/notifications.py:6]: or even better why that, can't we just use `j.core.tools. .` ?

File: ThreeBotPackages/zerobot/admin/actors/notifications.py (hunk 1 above)
Comment (waleedhammam) [inline ThreeBotPackages/zerobot/admin/actors/notifications.py:15]: also could be replaced with `j.core.tools.is_latest_release()`

File: ThreeBotPackages/zerobot/admin/actors/notifications.py [hunk 3]
```diff
@@ -8,4 +12,7 @@ def _init(self, **kwargs):
 
     @j.baseclasses.actor_method
     def check_new_release(self, schema_out=None, user_session=None):
-        return j.data.serializers.json.dumps(Tools.get_newest_version())
+        if not Tools.is_latest_release():
```
Comment (waleedhammam) [inline ThreeBotPackages/zerobot/admin/actors/notifications.py:15]: Can be replaced with `j.core.tools.is_latest_release()` and remove the import and try except

File: ThreeBotPackages/zerobot/admin/actors/notifications.py (hunk 2 above)
Comment (dmahmouali) [reply ThreeBotPackages/zerobot/admin/actors/notifications.py]: done :+1:  thanks waleed

File: ThreeBotPackages/zerobot/admin/actors/notifications.py (hunk 1 above)
Comment (dmahmouali) [reply ThreeBotPackages/zerobot/admin/actors/notifications.py]: yes, have to be `not is_latest_release()`, changed and thanks Jo

Comment (dmahmouali) [reply ThreeBotPackages/zerobot/admin/actors/notifications.py]: thanks Waleed :+1:

File: ThreeBotPackages/zerobot/admin/actors/notifications.py (hunk 3 above)
Comment (dmahmouali) [reply ThreeBotPackages/zerobot/admin/actors/notifications.py]: thanks waleed :+1:

**Pull request merged**